models/tasks.js

```diff
@@ -67,13 +68,13 @@ const fetchTasks = async () => {
 
 const fetchActiveTaskMembers = async () => {
   try {
-    const tasksSnapshot = await tasksModel.where('status', '==', 'active').get()
+    const tasksSnapshot = await tasksModel.where('type', '==', TASK_TYPE.FEATURE).where('status', '==', TASK_STATUS.ACTIVE).get()
     const activeMembers = []
     tasksSnapshot.forEach((task) => {
       const taskData = task.data()
-      if (taskData.participants) {
```

Comment:
Instead of extra if condition , I will be calling ```const {assignee} = task.data()``` here...
Would there be a possibility of occurrence of undefined in this case?

Reply:
What do the docs say?

Reply:
https://firebase.google.com/docs/reference/js/firebase.firestore.DocumentSnapshot#data
If the document doesn't exist it returns undefined 
We can push undefined to array (confirmed in mdn docs example).
Would it be better to stick to current plan when if the assignees exists then they are being added?

Reply:
What's an alternative you suggest?